client/lib/posts/post-list-store.js

```diff
@@ -204,8 +212,31 @@ export default function( id ) {
 
 		if ( newPostIds.length ) {
 			_activeList.postIds = _activeList.postIds.concat( newPostIds );
-			sort( _activeList.postIds );
+			_activeList.postIds = sort( _activeList.postIds );
+		}
+	}
+
+	/**
+	 * Find deleted posts based on ommissions in a new page of results.
+	 * In some cases the new list may overlap with our stored list. And when
+	 * this happens we can find elements that should be removed if they don't
+	 * appear in the new post-list.
+	 *
+	 * @param  {array} currentList - stored list of PostIDs
+	 * @param  {array} newPosts    - new page of postIDs
+	 * @return {array}             - postIDs in currentList that should be deleted
```

Comment:
I'm using docblockr (in atom) to set these, and it seems opinionated about syntax. Which aspect of @return are you taking issue with?